mentirinha/tasks.py

```diff
@@ -0,0 +1,7 @@
+from mentirinha import counter
+
+# TODO: vai ser uma periodic task isso aqui
+def consume_counter():
+    for url_id in counter.list_shortned_urls_ids():
+        counter = counter.getdel(url_id)
+        ShortenedUrl.objects.filter(pk=url_id).update(counter=F('counter') + 1)
```

Comment:
```python
counter=F('counter') + counter)
```

Reply:
E counter vai quebrar quando for null. Pode ter concorrência aí.

Reply:
Acho que essa coluna nao é nullable.

Nao entendi onde pode ter concorrencia. Se dois updates acontecerem juntos o valor final ainda vai ser o certo, certo?

Reply:
O getdel pode retornar null se não tiver mais chave lá.